### ISSUE TITLE
TinyMCE not displaying

### ISSUE THREAD

**Issue by TomGrozev** [daveroverts/bmac#160]
Hi,

I have set up the project as described and it works well except for the fact that the description field on the events page does not show up. It seems to be because it has visibility is hidden and the TinyMCE is not activating correctly. I have attached screenshots of a couple of things I noticed that seem to be related. I'm not all that familiar with Laravel otherwise I would have fixed it myself.

**Screenshots:**
Add/edit event page
![a](https://user-images.githubusercontent.com/1491414/97678447-e95ebe00-1ae7-11eb-8ff2-a67e0acbb96c.PNG)

DOM of descripion field
![b](https://user-images.githubusercontent.com/1491414/97678451-ea8feb00-1ae7-11eb-821f-967dcdca8a26.PNG)

Last tag in head
![c](https://user-images.githubusercontent.com/1491414/97678456-ec59ae80-1ae7-11eb-9b93-6371f5f7685b.PNG)

Any assistance would ve appreciated. Thanks

**Comment by daveroverts** [comment]
Thanks for the report. I indeed notice that TinyMCE doesn't want to load in on my staging environment. However, it works perfectly on my local machine. That's why I didn't notice it before.

For whoever reads this and wants to fix it (as I'm clueless at this point): All I can say is that on my staging env, it attempts to load in JS and CSS with the name 'undefined', even though it should load in oxide. I've already tried clearing cache both locally and on Cloudflare.
![Schermafbeelding 2020-11-09 om 10 39 47](https://user-images.githubusercontent.com/7283874/98525214-789e6b00-2278-11eb-8ebb-1b866ca7f7ab.png)
